### ISSUE TITLE
This extension cannot be trustworthy

### ISSUE THREAD

**Issue by SoniEx2** [shinigami-eyes/shinigami-eyes#44]
To anyone who comes across this issue: consider using uBlock Origin instead.

Not only does uBlock Origin work on mobile, but you have a lot more control over who you trust. As it currently stands, this extension is managed by an anonymous person and the list is maintained with no transparency whatsoever, the list doesn't get regularly updated, and it is impossible to mirror the list so as to make it fly under the radar so it automatically and immediately outs the extension user as trans-friendly. uBlock Origin supports properly community-managed, openly forkable, mirrorable lists, which doesn't put the list user at risk and doesn't centralize sensitive information. Additionally, you can even make your own lists and share them on github.

Please don't use this extension.

**Comment by solonovamax** [comment]
Yeah, it's just this extension really provides something that uBlock cannot.

But as for the public lists, I've made an issue (#45) for open-sourcing the backend and allowing people to host their own instances. If the maintainer doesn't reply, then I may consider just hard forking this repo and creating my own fork with the option to use your own instances.

**Comment by SoniEx2** [comment]
And what's that? This extension doesn't support arbitrary websites/domains, nor does it provide plausible deniability. In general you're far more likely to be arrested for using an extension like this one than for using something like uBlock Origin.

**Comment by solonovamax** [comment]
First of all, you can't really be arrested for using this extension. It does nothing illegal.
But next, this extension allows you to see both people who are pro-trans and anti-trans. Whereas a ublock list would just remove the anti-trans people from the page (at least, that's how I assume it'd work)
And although some people might say that the ability to show pro-trans people is bad, it depends on your perspective, That's why I want an open-source version of the database, so there could be instances that purposely disable the "pro-trans" flag.

**Comment by SoniEx2** [comment]
you can be arrested for being pro-trans sometimes. and using the extension probably makes you pro-trans (sometimes, anyway).

**Comment by solonovamax** [comment]
> you can be arrested for being pro-trans sometimes
that's why I said it depends on your perspective.
> and using the extension probably makes you pro-trans (sometimes, anyway).
It does not. Read the source code.

**Comment by ReticentAdjuvant** [comment]
> 
> 
> Yeah, it's just this extension really provides something that uBlock cannot.
> 
> But as for the public lists, I've made an issue (#45) for open-sourcing the backend and allowing people to host their own instances. If the maintainer doesn't reply, then I may consider just hard forking this repo and creating my own fork with the option to use your own instances.

Please do. I'm glad I had the sense to investigate this addon before installing it. I am constantly on the hunt for addons and the like that supplements my ability to identify and summarize the background of sources. It would be very appreciated if you were to develop something transparent to all. As an aside, I wouldn't be upset if you added reddit to the addon either.
